internal/testplanet/planet_test.go

```diff
@@ -37,21 +38,18 @@ func TestBasic(t *testing.T) {
 			t.Log("UPLINK", uplink.ID(), uplink.Addr())
 		}
 
-		// ping a satellite
-		_, err = planet.StorageNodes[0].Kademlia.Service.Ping(ctx, planet.Satellites[0].Local().Node)
+		sat := planet.Satellites[0].Local().Node
+		node := planet.StorageNodes[0].Local()
+		conn, err := planet.StorageNodes[0].Transport.DialNode(ctx, &sat)
 		require.NoError(t, err)
-
-		// ping a storage node
-		_, err = planet.StorageNodes[0].Kademlia.Service.Ping(ctx, planet.StorageNodes[1].Local().Node)
-		require.NoError(t, err)
-
-		err = planet.StopPeer(planet.StorageNodes[1])
+		_, err = pb.NewNodeClient(conn).CheckIn(ctx, &pb.CheckInRequest{
+			Address:  node.GetAddress().GetAddress(),
+			Version:  node.Version.GetVersion(),
+			Capacity: &node.Capacity,
+			Operator: &node.Operator,
+		})
 		require.NoError(t, err)
 
-		// ping a stopped storage node
-		_, err = planet.StorageNodes[0].Kademlia.Service.Ping(ctx, planet.StorageNodes[1].Local().Node)
-		require.Error(t, err)
-
 		// wait a bit to see whether some failures occur
 		time.Sleep(time.Second)
```

Comment:
can we remove this sleep? not sure what failures this is waiting for

Reply:
I'm unsure but I think it was used to just make sure nothing crashed in setup... @egonelbre do you know if it's still necessary?

Reply:
keeping this for now